rollbar.js

```diff
@@ -81,7 +81,7 @@ exports.init = function (accessToken, options) {
    *  
    */
   if (!initialized) {
-    if (!accessToken) {
+    if (!accessToken && options.enabled !== false) {
```

Comment:
Oh, right, I didn't think of that. The problem would be solved if I move 
```
options = options || {};
```
above the condition.

The reason for this condition is that in development mode I don't have a accessToken, and since its not an error when enabled = false the warning is redundant.

Should I remove it nevertheless? Or just fix the empty-options-case?